Automattic/wp-calypso
# Stats: Update traffic page layout

#### Proposed Changes

Reflows the Traffic page layout to adhere to the design specified in #68975. Specifically, this PR:
- Grids the modules list
- Grids the country module
- Flexboxes all stats modules
- Reorders module declarations within the modules list

#### Testing Instructions

* Spin up this change on a Calypso live branch.
* Navigate to the Traffic Stats page with `?flags=-stats/new-stats-module-component` appended to the URL to **mimic the current production interface**.
* Ensure that the page looks reasonable at all breakpoints. Note that the SVG map in the country module does not handle resizing well.
* (Bonus) Ensure that the Insights, Store, and Ads pages remain unaffected.
* (Bonus) Ensure that the "post details" and "post summary" pages remain unaffected.

#### Pre-merge Checklist



#### Screenshot

![calypso 2](https://user-images.githubusercontent.com/4044428/200967875-c0c9449d-9835-4a7b-adcd-eb993d6e32ea.png)
